mods/disable-ms-teams-hotkey.wh.cpp

```diff
@@ -0,0 +1,48 @@
+// ==WindhawkMod==
+// @id              disable-ms-teams-hotkey
+// @name            Disable Microsoft Teams hotkeys used by Visual Studio
+// @description     Disables Ctrl+Shift+Space that does something useful in Visual Studio and not in Teams.
+// @version         1.0.0
+// @author          Yves Goergen
+// @github          https://github.com/ygoe
+// @homepage        https://ygoe.de/
+// @include         ms-teams.exe
+// ==/WindhawkMod==
+
+// ==WindhawkModReadme==
+/*
+# Disable Microsoft Teams hotkeys used by Visual Studio
+
+Microsoft Teams occupies the keyboard shortcut Ctrl+Shift+Space to do whatever nonsense with it while sharing the screen.
+This hotkey is also assigned in Visual Studio by default to show the parameters info of the method under the cursor.
+That information is often very useful while coding, but it's not available in a team coding session when sharing the screen.
+So this mod prevents Teams from ever occupying that hotkey so that it remains available to Visual Studio like it's supposed to.
+*/
+// ==/WindhawkModReadme==
+
+BOOL(*pOriginalRegisterHotKey)(HWND hWnd, int id, UINT fsModifiers, UINT vk);
+
+BOOL RegisterHotKeyHook(HWND hWnd, int id, UINT fsModifiers, UINT vk)
```

Comment:
The calling convention is missing here, which is likely to cause a crash for 32-bit targets. Please add calling conventions or, if the mod doesn't support 32-bit, limit it to 64-bit, which will also make compilation faster. See:
https://github.com/ramensoftware/windhawk/wiki/Development-tips#hooks-and-calling-conventions

Reply:
This would also apply to the source I used: https://windhawk.net/mods/disable-feedback-hub-hotkey

Reply:
Right, it's a bug there too. I didn't notice it back then, now it's up to @ItsProfessional to fix it.

This issue [caused crashes in the past](https://github.com/ramensoftware/windhawk-mods/issues/136). For that mod, which only targets explorer.exe, it's less severe, since the 32-bit explorer.exe is only used in 32-bit systems which aren't common nowadays.